camelot/view/controls/delegates/floatdelegate.py

```diff
@@ -79,6 +95,26 @@ def get_standard_item(cls, locale, model_context):
             item.setData(py_to_variant(str()), PreviewRole)
         return item
 
-
+    def setEditorData(self, editor, index):
+        if index.model() is None:
+            return
+        self.set_default_editor_data(editor, index)
+        suffix = variant_to_py(index.data(SuffixRole))
```

Comment:
de variant to py is enkel nuttig bij datum waardes ... dus die omzettingen zijn hier in feite nt nodig